index.js

```diff
@@ -327,7 +332,11 @@ class ExecutorQueue extends Executor {
         Object.assign(newConfig, config);
 
         return this.postBuildEvent(newConfig)
-            .catch(() => Promise.resolve());
+            .catch((err) => {
+                winston.err(`failed to post build event for job ${config.job.id}: ${err}`);
```

Comment:
this should be `${config.jobId}`, the original config has `jobId` and `jobName` separately